docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,197 @@
+=========================================
+PostgreSQL specific aggregation functions
+=========================================
+
+.. module:: django.contrib.postgres.aggregates
+   :synopsis: PosgreSQL specific aggregation functions
+
+All of these functions are available
+from the ``django.contrib.postgres.aggregates``.
+
+.. warning::
+
+    All functions come without default alias,
+    so you must explicitly provide it.
```

Comment:
This note needs more explanation I think - an example should suffice. Also it should be `aliases`.